core/unit_test/TestTeamBasic.hpp

```diff
@@ -181,6 +181,8 @@ struct LargeTeamScratchFunctor {
   }
 };
 
+#if !KOKKOS_32_BIT // FIXME: This fails on 32-bit atm.
```

Comment:
This should instead be a `GTEST_SKIP() << "Fails on 32-bit";` inside the test, so that we can see it in the output.

Reply:
Done also.